static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -79,12 +79,54 @@ define([
 
     function onRichLinksUpgraded(body) {
         return qwery('.element-rich-link--not-upgraded', body).length === 0 ?
-            Promise.resolve(true) :
+            true :
             new Promise(function (resolve) {
                 mediator.once('rich-link:loaded', resolve);
             });
     }
 
+    function onInteractivesLoaded(body) {
+        var notLoaded = qwery('.element-interactive', body).filter(function (interactive) {
+            var iframe = qwery(interactive.children).filter(isIframe);
+            return !(iframe.length && isIframeLoaded(iframe[0]));
+        });
+
+        return notLoaded.length === 0 ?
+            true :
+            Promise.all(notLoaded.map(function (interactive) {
+                return new Promise(function (resolve) {
+                    new MutationObserver(function (records, instance) {
```

Comment:
I'll count this as a progressive enhancement, in which case I should probably add a test for `MutationObserver`. Thanks for the shout out!